docs_src/microservices/support/scheduler/Ch-Scheduling.md

```diff
@@ -36,3 +24,31 @@ in your current configured database for EdgeX.
   |Interval        | An object defining a specific "period" in time.|
   |IntervalAction  | The action taken by a Service when the Interval occurs.|
   
+## Configuration Properties
+
+The following are extra configuration parameters specific to the Support-Scheduler service. Please refer to the general Configuration documentation for configuration properties common across all services.
```

Comment:
Make general Configuration documentation a link back to the general Configuration documentation